clang/test/SemaSYCL/unique-stable-id.cpp

```diff
@@ -101,9 +101,6 @@ void invalidated() {
     // lambda.
     static int GlobalStorageVar;
     constexpr const char *c = __builtin_sycl_unique_stable_id(GlobalStorageVar);
-    // expected-error@+2{{kernel naming changes the result of an evaluated '__builtin_sycl_unique_stable_id'}}
-    // expected-note@-2{{'__builtin_sycl_unique_stable_id' evaluated here}}
-    __builtin_sycl_mark_kernel_name(K);
   };
```

Comment:
Is this part of the test still useful?

Reply:
I don't believe so, I'll delete this part of the test.